### DISCUSSION TITLE
转账交易提示Capacity not enough!

### DISCUSSION THREAD

**Discussion by chaim-g** [nervosnetwork/ckb#2402] [General]
你好！打扰下，我们这边在调用send_transaction方法时一直提示钱不够（实际上地址中是有足够钱的），直接转账一直是成功的，未作任何代码修改的情况下，现在调用总是不成功，报错位置是在CellCollectorWithIndexer类的collectInputs方法中。cell的阈值一直设置的是61个ckb。我转账63个币时 这个方法输出inputsCapacity = 0  needCapacity = 12300000331。再往下就抛异常了。

**Comment by doitian** [comment]
请问是用的什么工具构造交易的？
